### PR TITLE
update managing data section

### PR DESCRIPTION
**Pull request by ShahAnuj2610**
- [x] Importing from the Dashboard
- [x] Importing through ABC
- [x] Importing via REST API
- [x] Import via Zapier

### CODE REVIEW

File: content/docs/data/Import.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+
+### Upload the file
+
+Now that the app has been created, we can import the data file as shown in the GIF.
+
+![](https://www.dropbox.com/s/05do1q1wvp7t13t/short_upload.gif?raw=1)
+
+---
+
+##Importing through ABC CLI
```
Comment (siddharthlatest) [inline content/docs/data/Import.md:97]: @ShahAnuj2610 you're missing adaptors. See here for a list of all: https://github.com/appbaseio/abc/tree/dev/docs/importer/adaptors.

File: content/docs/data/Import.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+     </a>
+	<a class="bg-white shadow-2 box-shadow-hover shadow-2-hover  br4 db flex flex-column justify-between items-center middarkgrey pa2 pt5 pb5 tdn tc" style="box-shadow: 0 0 5px rgba(0,0,0,.02), 0 5px 22px -8px rgba(0,0,0,.1);    word-break: normal;cursor: pointer; padding: 2rem; height: 120px;width:120px;" href="#zapier">
+		<img class="w10 mb1" src="https://i.imgur.com/Z9ahZbM.png" />
+		Zapier
+	</a>
+</div>
+
+---
+
+##Importing through Dashboard
```
Comment (siddharthlatest) [inline content/docs/data/Import.md:40]: @ShahAnuj2610 We can show the dashboard importer screen UI at the start of the section here.

File: content/docs/data/Import.md [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+
+```bash
+abc import --src_type=firestore --sac_path=<path_to_service_account_credentials> <elasticsearch_uri>
+```
+
+Read more about it over [here](https://medium.appbase.io/cli-for-indexing-from-firestore-to-elasticsearch-80286fc8e58b)
+
+---
+
+##Rest API
```
Comment (siddharthlatest) [inline content/docs/data/Import.md]: s/Rest/REST

File: content/docs/data/Import.md [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+This is just getting started with Rest API. You can read more about it over [here](https://rest.appbase.io/).
+
+### Setup
```
Comment (siddharthlatest) [inline content/docs/data/Import.md]: @ShahAnuj2610 This section should talk about the indexing part only. It should mention the use of PUT / POST endpoint for creating / updating a document and the bulk API endpoint and how to use them.

File: content/docs/data/Import.md [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+	"department_name_analyzed": "Books",
+	"department_id": 1,
+	"name": "A Fake Book on Network Routing"
+}'
+```
+
+---
+
+##Zapier
+You can import data into appbase.io using any of the following methods:
```
Comment (siddharthlatest) [inline content/docs/data/Import.md]: @ShahAnuj2610 Why is the content here repeating?

Comment (siddharthlatest) [inline content/docs/data/Import.md]: @ShahAnuj2610 You can add the Zapier <> Appbase banner image.

File: content/docs/data/Import.md [hunk 6]
```diff
@@ -0,0 +1,319 @@
...
+![](https://i.imgur.com/r2MSpTg.png)
+
+### Updating An Existing Document
+
+Another helpful feature is the ability to edit existing documents. Whenever a value from an incoming integration is mapped to an `_id` field, the existing document gets updated instead of a new document getting created.
+
+![](https://i.imgur.com/7zEAso9.png)
+
+### Conclusion
+
```
Comment (siddharthlatest) [inline content/docs/data/Import.md:490]: @ShahAnuj2610 There should be an overall conclusion of the four methods and a recommendation of when to use each method. We can use a table to show this.

File: src/styles/custom.css [hunk 7]
```diff
@@ -1826,12 +1826,12 @@ to be custom HTML.
 }
 
 .truncate-3 {
-    line-height: 1.4;
-    display: -webkit-box;
-    -webkit-line-clamp: 3;
-    -webkit-box-orient: vertical;
-    overflow: hidden;
-    text-overflow: ellipsis;
+	line-height: 1.4;
```
Comment (siddharthlatest) [inline src/styles/custom.css:1829]: @ShahAnuj2610 Why are mixing spaces and tabs indent?

**Review by siddharthlatest — CHANGES_REQUESTED** [review]

File: src/styles/custom.css (hunk 7 above)
Comment (ShahAnuj2610) [reply src/styles/custom.css:1829]: it's due to prettier

**Pull request merged**